content/en/apps/examples/contact-tracing.md

```diff
@@ -34,8 +34,10 @@ Contact Tracing with the CHT offers a rapidly scalable surveillance strategy by
 
 ![hierarchy.png](hierarchy.png)
 
-## App Workflow Example
+## Workflow Example
 
-This is a detailed workflow that supports tracing of contacts of suspected or confirmed patient cases, symptom screening, care referrals, and monitoring of asymptomatic contacts during self-isolation. It works on both SMS and the CHT app. It can be combined with other care workflows and configured to suit specific health system needs, particularly in terms of tracer roles. 
+This demo illustrates how a CHT workflow for tracing of contacts of suspected or confirmed patient cases can be deployed to support a COVID-19 health program response. It works on both SMS and the CHT app. It can be combined with other care workflows and configured to suit specific health system needs, particularly in terms of tracer roles. 
 
-![workflow.png](workflow.png)
+<iframe width="560" height="315" src="https://www.youtube.com/embed/I8bBeh80j-0" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
We should use the youtube shortcode for this, and avoid autoplay -- both of which should be added to our docs style guide! :)

```suggestion
{{< youtube I8bBeh80j-0 >}}
```